.github/workflows/backport.yml

```diff
@@ -0,0 +1,33 @@
+name: Backport Merged Pull Request
+
+on:
+  pull_request:
+    types:
+      - closed
+      - labeled
+
+jobs:
+  backport:
+    name: Backport pull request
+    runs-on: ubuntu-latest
+    if: >
+      github.event.pull_request.merged
+      && (
+        github.event.action == 'closed'
+        || (
+          github.event.action == 'labeled'
+          && contains(github.event.label.name, 'backport')
+        )
+      )
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+    - name: Run custom bash script for backporting
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        GITHUB_EVENT: ${{ toJson(github.event) }} # Pass the github.event object as a JSON string
+      run: |
+        cd $GITHUB_WORKSPACE  # Move to the root of the repository
+        cd .github/scripts/backport
```

Comment:
Why cd into the scripts directory when you can `./.github/scripts/backport/main.sh` instead?

Reply:
I tried using `./.github/scripts/backport/main.sh` first in my repo but somehow it was not able to locate the directory.

Reply:
updated.